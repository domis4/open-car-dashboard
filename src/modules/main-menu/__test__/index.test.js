import React from 'react'
import { shallow } from 'enzyme'
import MainMenu from '../index'

describe('main-menu module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<MainMenu />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
