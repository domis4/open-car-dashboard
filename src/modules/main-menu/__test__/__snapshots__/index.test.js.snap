// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main-menu module index.js expects component to match snapshot 1`] = `
<WithStyles(Drawer)
  classes={
    Object {
      "paper": "View-drawerPaper-04",
    }
  }
  variant="permanent"
>
  <WithStyles(List)
    className="View-top-01"
  >
    <WithStyles(ListItem)
      button={true}
      className="View-center-06"
      onClick={[Function]}
    >
      <pure(ArrowBackIosIcon) />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      button={true}
      className="View-center-06"
      onClick={[Function]}
    >
      <pure(FilterNoneIcon) />
    </WithStyles(ListItem)>
  </WithStyles(List)>
  <WithStyles(List)
    className="View-bottom-03"
  >
    <WithStyles(ListItem)
      className="View-center-06"
    >
      <withState(lifecycle(WithStyles(View))) />
    </WithStyles(ListItem)>
    <WithStyles(ListItem)
      button={true}
      className="View-center-06"
      onClick={[Function]}
    >
      <div
        className="View-apps-07"
      />
    </WithStyles(ListItem)>
  </WithStyles(List)>
</WithStyles(Drawer)>
`;
